config/v1/types_cluster_version.go

```diff
@@ -224,6 +233,16 @@ type UpdateHistory struct {
 // ClusterID is string RFC4122 uuid.
 type ClusterID string
 
+// ClusterVersionArchitecture enumerates valid cluster architectures.
+// +kubebuilder:validation:Enum=Multi
```

Comment:
Enum has to allow "" in order to handle a golang json serialization.

Reply:
Fixed in https://github.com/openshift/api/commit/32939f536c1b41128f58f87a26534f3bca036359